### PR TITLE
[tools] add clickbench tools

### PR DESCRIPTION
**Pull request by hello-stephen**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: tools/clickbench-tools/load-clickbench-data.sh [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+    if ${exit_flag}; then exit 1; fi
+}
+
+function load() {
+    echo "(1/2) prepare clickbench data file"
+    if [ ! -f $DATA_DIR/hits.tsv.gz ]; then
+        echo "can not find data file in local, will download it"
+        if [ ! -d $DATA_DIR ]; then mkdir -p $DATA_DIR; fi
+        cd $DATA_DIR && wget --continue 'https://datasets.clickhouse.com/hits_compatible/hits.tsv.gz' && cd -
```
Comment (yiguolei) [inline tools/clickbench-tools/load-clickbench-data.sh:134]: 把这个放到我们自己的COS 上吧，要不下载太慢了

File: tools/clickbench-tools/load-clickbench-data.sh [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+ROOT=$(dirname "$0")
+ROOT=$(
+    cd "$ROOT"
+    pwd
+)
+
+CURDIR=${ROOT}
+DATA_DIR=$CURDIR/
+# DATA_DIR=/mnt/disk1/stephen/data/clickbench
```
Comment (liaoxin01) [inline tools/clickbench-tools/load-clickbench-data.sh:33]: delete this line?

**Review by kpfly — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**